ci-operator/config/openstack-k8s-operators/openstack-k8s-operators-ci/openstack-k8s-operators-openstack-k8s-operators-ci-main.yaml

```diff
@@ -49,6 +49,24 @@ images:
       as:
       - registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.18-openshift-4.12
   to: ci-build-root-golang-1.19-sdk-1.31
+- build_args:
+  - name: SDK_VERSION
+    value: v1.31.0
+  dockerfile_path: images/build-root/Dockerfile.custom
+  inputs:
+    rhel-8-golang-1.20:
```

Comment:
this need to be defined in base_images, at the top of this file:
e.g.:
```
rhel-8-golang-1.19:
    name: release
    namespace: openshift
    tag: rhel-8-release-golang-1.19-openshift-4.12
```

Reply:
For golang 1.20 we can use the tag:
`"rhel-9-release-golang-1.20-openshift-4.14"`
and for 1.21:
`"rhel-9-release-golang-1.21-openshift-4.16"`